lib/Conversion/TTIRToTTNN/TTIRToTTNN.cpp

```diff
@@ -871,6 +874,77 @@ class Conv2dOpConversionPattern : public OpConversionPattern<ttir::Conv2dOp> {
   }
 };
 
+class ConvTranspose2dOpConversionPattern
```

Comment:
Please add a brief descrpipiton of conversion as it isn't 1-1 mapping to ttnn conv2d.